cmdline-pull/pubsub_sample.py

```diff
@@ -104,12 +74,19 @@ def list_topics(client, args):
 
 
 def list_subscriptions(client, args):
-    """Shows the list of current subscriptions."""
+    """Shows the list of current subscriptions attached to a given topic, or all
```

Comment:
Keep the summary line short, and add details after one blank line.
See: https://www.python.org/dev/peps/pep-0257/